docs/UserGuide.md

```diff
@@ -174,6 +174,40 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd person in CulinaryContacts.
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
+### Archive
+
+#### Archiving a person: `archive`
+Removes the specified person from the contacts list and adds the person into the archived list.
+
+Format: `archive INDEX`
+
+* Archives the person at the specified INDEX.
+* The index refers to the index number shown in the _displayed person list_. 
+* The index must be a positive integer 1, 2, 3, ...
```

Comment:
I think it's better to bold the **must be a positive integer** as what existing commands info have.